gwpy/table/tests/test_table.py

```diff
@@ -505,15 +555,27 @@ def test_read_pycbc_live(self):
             utils.assert_table_equal(table, t2)
 
             # assert processed colums works
-            t2 = self.TABLE.read(fp, ifo='X1', columns=['mchirp', 'new_snr'])
+            t2 = self.TABLE.read(fp,
```

Comment:
@duncanmmacleod, PEP-8 is complaining about this line, maybe to pacify it you can make the following change?

```suggestion
            t2 = self.TABLE.read(
                fp,
```